### DISCUSSION TITLE
Fisheries data

### DISCUSSION THREAD

**Discussion by pyhernvann** [pfmc-assessments/shortspine_thornyhead_2023#4] [General]
Fisheries landings & length/age compo discussion!

**Comment by JaneSullivan-NOAA** [comment]


**Reply by haleyoleynik** [reply]
Yes, I can meet anytime within that window!

**Reply by shipmadison** [reply]
I could meet at 3:30, but have to be done by 4:15 for another obligation!

**Comment by pyhernvann** [comment]
Hey
I can't join before 3.20 PM but happy to meet
PY

Le ven. 3 févr. 2023 à 11:48, Haley Oleynik ***@***.***> a
écrit :

**Comment by pyhernvann** [comment]
@haleyoleynik could you push once you're done so that I can work with your latest version of the code before Thursday?

**Reply by haleyoleynik** [reply]
@pyhernvann - [done](https://github.com/JaneSullivan-NOAA/shortspine_thornyhead_2023/blob/main/R/SST_PacFIN_expansion.R)!

**Reply by pyhernvann** [reply]
Thanks @haleyoleynik

**Comment by pyhernvann** [comment]
(I've just pushed an updated SST_Pacfin_expansion file with a magical  correction for integrating W-L outliers)

**Comment by haleyoleynik** [comment]
@pyhernvann @Ovec8hkin @shipmadison - I put in a [request for the longspine thornyhead landings data](https://github.com/pfmc-assessments/PacFIN.Utilities/issues/92). Based on the shortspine data request it looks like it doesn't matter who puts it in because Kiva will get it. Just sharing it here as an FYI.

**Comment by haleyoleynik** [comment]
### Fishery Landings 
So I took a look at the ratio of shortspine to total thornyheads ([WIP code here](https://github.com/JaneSullivan-NOAA/shortspine_thornyhead_2023/blob/eeec690f27025de4a408c75a0d9d01b335c1e149/R/landings/SST_PacFIN_Fishery_Landings_2023.R#L59)). It looks like the ratio differs by state depending on year ([see this figure in the slides](https://docs.google.com/presentation/d/1tJ0B4Ch14V3xKqTK0jz_4sOj8VQli1NA7fQ38SRBRHw/edit#slide=id.g1b63de32ef5_0_0)). The 2013 assessment didn't give any justification about why they took the coastwide ratio, but I'm sure they had some reason.. maybe worth asking... Anyway, wanted to get all of your thoughts on which method we should use, if anyone has any. Seems like based on that figure we should use the state-specific ratio, but there could be a problem with that I haven't yet thought through. And of course, we can always throw in both as another sensitivity analysis!

@Ovec8hkin @pyhernvann @shipmadison

**Reply by Ovec8hkin** [reply]
Based on the figure, it seems like using a state-specific ratio would be a good idea. The next questions becomes, should we take the long-term average ratio and apply it to all years, or use the state-and-year-specific ratio to assign the unidentifieds?

**Reply by haleyoleynik** [reply]
I think definitely the state-and-year specific ratio - it seems to have changed a lot through time probably in part a function of where they fish. 

So: we split unID'd thornyhead catches by state and year, apply the ratio (shortspine / total), and then add that in to shortspine catches. 

I wonder if it's also worth looking at the ratio by gear type ... I may be going too far down the rabbit hole now, though.

**Reply by pyhernvann** [reply]
Thanks for this @haleyoleynik - The trend shared across states is meaningful.
I don't have a very deep-seated opinion on aggregation level we should use (US vs States). On the one hand, the differences in the absolute values may be interesting to account for. On the other hand it remains difficult to assess the relative level of confidence in SST/LST distribution inferred from this (e.g. pertinence regarding the whole fishery?; regarding the sampling intensity, are we sure that these differences in the absolute values are ), and the 1995 shift in Washington ratio is a good illustration that the way that these ratios are calculated in the different states may have an impact on the . 
Just thoughts - I'm not sure it helps

**Reply by pyhernvann** [reply]
* just seeing your last message:
- yes, let's consider year specific ratios!
- No, let's try not to fall into this rabbit hole of investigating gears
- questions: (1) early ratio for WA looks unrealistic but maybe it's ok due to the total landings in WA? (2) should directly use these ratios or a running mean (or any simple tool) to smooth a little bit these trends?

**Reply by haleyoleynik** [reply]
@pyhernvann in response to the unrealistic Washington ratio - I just realized WA has no unidentified thornyhead catches - so this won't matter!

**Reply by adamlhayes** [reply]
Everyone keeps saying that the Oregon and Washington vessels fish the same areas, so it makes sense to collapse those two states at least. I would guess there's significant differences by gear type. So, it makes sense to me to allocate the unspecified Thornyheads by the year-fleet ratio, where the fleets are the four that have typically been used.

**Comment by pyhernvann** [comment]
Hi fisheries data team - especially @haleyoleynik and @Ovec8hkin who are not part of the discards team
We got an answer from John Wallace for the Pikitch discard data. He's able to generate the discard rates + length compo for us. He also ask us to choose between this data with or without sex information (he doesn't seem to be willing to generate both).

Just checking your opinion about this. So far, the discard team and I quickly talked about it and found that it may not be useful as (i) we don't have sex info in WCGOP data [we didn't considering requesting it and I don't know if it is possible to access it; question for tomorrow..?] (ii) Pikitch = only three old years (1985-1987) and one fleet (N Trawl) (iii) fitting the model selectivity for these 3 years and 2 sex may be not parsimonious and last (iv) it was not identified at all as a need/priority in the previous assessment and I don't remember from our data exploration that a substantial difference between sex was observed in the landings compo

Let me know - potentially a question to ask tomorrow

PY

**Comment by haleyoleynik** [comment]
### Fishery length comps 
Just wanted to post the status of fishery length comps here so we're all on the same page (and can give it some thought/ discuss in today's meeting) - our issue was that in WA in 2021 and 2022, the expansion factor for a bunch of samples was being capped at 1 ([see this diagnostic](https://github.com/JaneSullivan-NOAA/shortspine_thornyhead_2023/blob/main/outputs/fishery%20data/PacFIN_exp1.png)). Upon further digging, it looks like a 209 samples (fish) from 19 trips are missing total weights (`RWT_LBS` or `TOTAL_WGT`), which is why those are being capped. 

The instructors said not to worry so much about the 2022 trips - sometimes fish ticket info takes a while to get into PacFIN. Theresa from WDFW said to cross reference the fish ticket ID (`FTID`). 
- I did this, and there appears to be 5 trips in 2021 which do have total weights (`LANDED_WEIGHT_LBS` and `ROUND_WEIGHT_LBS` in the catch data) in the PacFin catch data, but not in the bio data). The difference in names of these variables makes it a bit confusing, but it seems to me that these vars would match `RWT_LBS` or `TOTAL_WGT` in the bio data. 
- The question is - do we need to worry about these 5 trips. If so, can we use the total weight from the catch data in the bio data for the first expansion?

**Reply by JaneSullivan-NOAA** [reply]
thanks @haleyoleynik! let's ask the instructors today in class. i have time this afternoon to work on this if you'd like... we can push through and get some preliminary length comps constructed for the data mtg.

**Reply by pyhernvann** [reply]
thank you @haleyoleynik for delving into it.
I'll have a look at these columns next Monday to just see if I reach the same conclusion. If there is no immediate solution, we can exclude them for our preliminary model runs and go back to it in a couple of weeks, checking whether the it got into PacFIN..?

**Reply by JaneSullivan-NOAA** [reply]
@haleyoleynik and @pyhernvann - okay I asked this question in class today and Vlada says that if we can match the samples by fish ticket number then it's absolutely okay to use the total weight from the catch data in the bio data for the first expansion

**Comment by haleyoleynik** [comment]
### Historical landings

From the 2013 assessment: 

> Catches prior to 1981 were set equal to those used in the previous model, rather than to the reconstructed history provided by CDFW and ODFW for most West Coast assessments. The California catch reconstruction did not split unspecified thornyheads into the two species. Furthermore, the recordings of longspine thornyhead prior to 1981 (e.g. 0.2 mt in 1977) are so low that the ratio of specified catch is not likely to be representative of the true ratio. The impact on the shortspine assessment of assuming all pre- PacFIN catch was shortspine is smaller than the impact of this assumption on the longspine assessment, but using the catch reconstruction for one species and the values from the previous assessment for another would risk double counting catch. Therefore, the catch from the previous assessments, both of which had a thorough independent review, were used for both species in the current assessments for the years prior to 1981. A sensitivity analysis indicated that the differences in these alternative assumptions about historical catch had very little impact on the model results (Figure 62, Figure 63).

@adamlhayes, forgive me if I totally missed a discussion on this - but do we have this problem with the new reconstructions for OR and WA? Did anyone mention how they handled unidentified thornyheads?

@pyhernvann @Ovec8hkin @JaneSullivan-NOAA @shipmadison

**Reply by adamlhayes** [reply]
WA and OR state data is split out by species, but California is not in their latest reconstruction from Ralston in 2010. There is CALCOM data starting in 1969 for California, and that has Shortspine, Longspine, and unidentified thornyhead if we want to try to use that. Or, we can use the STrawl and SOther catch reconstructions from the 2005 and 2013 assessments.

**Comment by adamlhayes** [comment]
Now that I've received WA and OR data, I summarized catch by fleet for the time series data. For OR these data go through 2022, and for WA through 2014. Isolating the pre-2010 to make sure all the state data is there, there are some considerable differences in landings totals for the 1981-1986 period for the northern trawl fleet across the data sources we have: state, PacFIN, and 2013 assessment. Looks like this is mostly because the OR time series in PacFIN doesn't begin until 1987. The WA state data is a little higher than the values from PacFIN as well, though not the same magnitude. Do we want to use state-level data for WA and OR until 1987, then switch to PacFIN, @haleyoleynik , @JaneSullivan-NOAA , @Ovec8hkin , @shipmadison ?
![image](https://user-images.githubusercontent.com/18746155/224448228-a713f4b6-ef3f-4096-bc1e-1afede299bba.png)

**Reply by JaneSullivan-NOAA** [reply]
Thanks @adamlhayes! Presumably PacFIN would be updated at some point to be consistent with the state reconstructions, but I'm not sure. Seems like a good thing to check on with the instructors.

**Reply by okenk** [reply]
I would defer to the state reps on this:

I think @aliwhitman (ODFW) prefers prefers to provide STATs with the full catch time series to use, so if she gave you through 2022, I would not use PacFIN data at all. Thank you Ali, and correct me if I am wrong!

@tsoutt (WDFW), the historical WA catches go through 2014. For what years should we use those vs data from PacFIN, for shortspine thornyhead? It sounds like this is most relevant for the 1980s.

I also want to flag that even the state catches are lower than values used in the 2013 assessment for those years in the 80s. @Ovec8hkin, do you know why?

**Reply by tsoutt** [reply]
For WA, it is OK to use PacFIN data for 1981 and forward. WDFW will not object to that approach. The use of WA catch reconstruction of the URCK market category for 1981-1999 period is inconsistent among stock assessments. There are also some other issues that we need to iron out with PacFIN for that time period. We will try to work these out for the next assessment cycle.

**Comment by aliwhitman** [comment]
Yes, I like to provide the whole times series all at once- partly from habit (in the past, some assessments have missed including nominal species landings, though that is really no longer an issue) but also because one of our historical reconstructions replaces some of the data in certain market categories in PacFIN during some years in the 80s and 90s, and it is much easier to have me combine those properly. 

With regards to the differences in 1980 - 1986 above, the past assessment did not have the Karnowski et al. 2014 (Oregon's historical commercial reconstruction) available. PacFIN has some strange expansions during that time period, at least for OR, so once that reconstruction was complete, we have always recommended using the Karnowski through 1986 and then PacFIN from 1987 forward. So that may explain some of the differences between the last assessment and the current time series. I'm not sure why PacFIN would be so low during this time period though. Are you including the nominal species as well?  

We would LOVE to update PacFIN with our reconstructions and have a true one stop shop there but it's been a slow process...

**Comment by adamlhayes** [comment]
One more question that we can discuss at the next landings meeting is how to handle California historical catch. I finally received the historical data from CA which is now posted on Github in the processed data folder. The data comes from two sources: CALCOM and the Ralston 2010 reconstruction. The issue is that almost all of the thornyhead landings before 1978 are not broken out into Shortspine and Longspine. I made a plot showing catch if the species group is split based on 1978-2022 fleet-specific averages (red line), if the entire species group is assigned to Shortspine (blue line), and the 2013 assessment values (black line). In addition, the Ralston reconstruction, which covers 1934-1969 does not differentiate by gear type. I assigned all catches to the trawl fleet. Both the red and blue lines include the foreign fleet catch from the Rodgers 2003 reconstruction, which the 2005 and 2013 assessments also use. 
![image](https://user-images.githubusercontent.com/18746155/224842446-c346faec-7898-4589-b3eb-476f3c99762e.png)

**Reply by okenk** [reply]
We should also have some time to talk about this in class this week, and I also think this would be a good topic to bring up at the presentation on Monday as there will be state reps who might have good thoughts on how best to handle this. 

In general, I think when the historical catch time series diverge from the 2013 assessment, it is important to be able to explain why. For instance, do you know why the 2013 assessment includes a bunch of catches from the 1940s-60s that are not in the Ralston reconstruction? Total fishery removals can have a big impact on stock status.

**Reply by JaneSullivan-NOAA** [reply]
thanks @adamlhayes and @okenk 

the CA catch reconstruction provided for the 2013 assessment also did not split between SST and LST, and the authors used this as justification to use catch estimates from the previous assessment. From the 2013 assessment:

> The California catch reconstruction did not split unspecified thornyheads into the two species. Furthermore, the recordings of longspine thornyhead prior to 1981 (e.g. 0.2 mt in 1977) are so low that the ratio of specified catch is not likely to be representative of the true ratio. The impact on the shortspine assessment of assuming all pre-PacFIN catch was shortspine is smaller than the impact of this assumption on the longspine assessment, but using the catch reconstruction for one species and the values from the previous assessment for another would risk double counting catch. Therefore, the catch from the previous assessments, both of which had a thorough independent review, were used for both species in the current assessments for the years prior to 1981. A sensitivity analysis indicated that the differences in these alternative assumptions about historical catch had very little impact on the model results (Figure 62, Figure 63).

The historical landings in the southern area (specifically STrawl) in the 2005/2013 assessments were calculated as 30% of the reconstructed sablefish trawl catch. From the 2005 assessment:

> To reconstruct estimated landings in the southern area before 1962, we compared recent domestic  catches of sablefish and shortspine thornyhead, with shortspine averaging about 44% the catch of  sablefish from 1962 through 2004. We assumed shortspine virtual landings to be 30% of the  reconstructed catch series for sablefish in the trawl fishery from 1901 to 1961 (M. Schirripa, pers.  comm.) (Table 3).

So I guess I'm curious if this is still the best available catch estimate for sablefish during that time period? And if so, are we all right with the 30% assumption?

In terms of the SST_split vs SST_all... I think the assumption you made (using the fleet-specific mean ratio of SST/(LST+SST) from 1978-2022) is reasonable, though there are some odd trends in that time series:

![image](https://user-images.githubusercontent.com/68303089/225072955-a96ff95b-7876-4008-b74d-59c1d251038f.png)

**Comment by okenk** [comment]
If you want to check, the last sablefish benchmark assessment is [here](https://www.pcouncil.org/documents/2019/10/status-of-the-sablefish-stock-in-u-s-waters-in-2019-october-22-2019.pdf) and the associated model files are [here](https://www.pcouncil.org/documents/2019/10/sablefish-model-files-october-2019.zip).

**Reply by JaneSullivan-NOAA** [reply]
thanks Kiva!

**Comment by haleyoleynik** [comment]
### Fishery landings length comps 

Hi all! Another discussion thread I wanted to start about something I noticed when I was doing the data write up - the 2013 assessment combined sexes for fishery length comps. From the 2013 assessment: 

> All length data from commercial fisheries included in the model with sexes combined. This avoids the possibility of bias due to difficulty in sex determination of thornyheads (also see notes below on sex ratios in survey data).

There obviously isn't a ton of justification here... So I'm wondering what you all think. @JaneSullivan-NOAA are thornyheads particularly hard to sex? 

@pyhernvann @Ovec8hkin @shipmadison

**Reply by JaneSullivan-NOAA** [reply]
Good catch! I didn't see that when I read the assessment. To the best of my knowledge, only small/immature thornyheads are difficult to sex. It might be worth following the same assumption as the last assessments... given the minimal differences in growth between the two sexes I don't really think there is a strong justification to assume separate selectivities anyway.
